src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -946,7 +963,25 @@ export const selectPortfolioAssetIdsByAccountIdExcludeFeeAsset = createDeepEqual
   },
 )
 
-export const selectAccountIdsByAssetId = createSelector(
+export const selectAccountIdByAddress = createSelector(
+  selectAccountIds,
+  selectAccountIdParamFromFilter,
+  (portfolioAccounts: { [k: AccountSpecifier]: AccountId[] }, filterAccountId): string => {
+    let accountSpecifier = ''
```

Comment:
If we are keeping this selector, can we kill the mutation?

Reply:
Nooked in 783fea474874bd366fd0361109e503cd54b9ab15